packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -787,16 +798,19 @@ export class RemotedFluidDataStoreContext extends FluidDataStoreContext {
         };
     });
 
-    private readonly gcDetailsInInitialSummaryP = new LazyPromise<IGarbageCollectionSummaryDetails>(async () => {
-        return (await this.getBaseSummaryGCDetails()) ?? {};
-    });
-
     protected async getInitialSnapshotDetails(): Promise<ISnapshotDetails> {
         return this.initialSnapshotDetailsP;
     }
 
+    /**
+     * @deprecated - Renamed to getBaseGCDetails.
+     */
     public async getInitialGCSummaryDetails(): Promise<IGarbageCollectionSummaryDetails> {
-        return this.gcDetailsInInitialSummaryP;
+        return this.getBaseGCDetails();
```

Comment:
It looks like you can collapse getInitialGCSummaryDetails & getBaseGCDetails symbols into one, given that they are doing same thing

Reply:
I have to keep both for back-compat of the runtime-definitions layer. There are validations that fail if I remove getInitialGCSummaryDetails. I will have to do that couple of versions later.